.github/workflows/ci-pytest.yml

```diff
@@ -0,0 +1,32 @@
+name: CI Pytest
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        # Python 3.9 not supported until next scancode-toolkit version
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Configure Plugin and Install Dependencies
+        run:  ./configure
+
+      - name: Activate Virtualenv
+        run: source tmp/bin/activate
+
+      - name: Run tests
+        run: tmp/bin/pytest -vvs
```

Comment:
Since you call a path here there is no need to activate above.

Reply:
Yes, changes made.